library/Rules/StringVal.php

```diff
@@ -13,8 +13,15 @@
 
 namespace Respect\Validation\Rules;
 
-class StringVal extends AbstractRule
+/**
+ * @author Danilo Correa <danilosilva87@gmail.com>
```

Comment:
Would be nice to have a single-line documentation in this docblock (you can find it in the documentation of the rule - the Markdown file)